src/libraries/Common/src/System/Net/Http/aspnetcore/Quic/Implementations/MsQuic/MsQuicConnection.cs

```diff
@@ -243,12 +246,28 @@ internal override ValueTask ConnectAsync(CancellationToken cancellationToken = d
         {
             ThrowIfDisposed();
 
+            (string address, int port) = _remoteEndPoint switch
+            {
+                DnsEndPoint dnsEp => (dnsEp.Host, dnsEp.Port),
+                IPEndPoint ipEp => (ipEp.Address.ToString(), ipEp.Port),
+                _ => throw new Exception($"Unsupported remote endpoint type '{_remoteEndPoint.GetType()}'.")
+            };
+
+            // values taken from https://github.com/microsoft/msquic/blob/main/docs/api/ConnectionStart.md
+            int af = _remoteEndPoint.AddressFamily switch
+            {
+                AddressFamily.Unspecified => 0,
```

Comment:
At least on Windows, these values should all be the same. Are they different on Linux? If so, a comment explaining this would be good.

Or is the point here just to exclude anything other than these 3?

Reply:
I don't believe they have defined values on Linux, so I did that.